From af0c03e3cd07ba8be372a3e6146d6cf72a1796a5 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Tue, 14 Mar 2017 11:23:55 +1000
Subject: [PATCH 4/6] test: don't use the same mouse twice

No effect since we don't care about the mouse itself. But when running
on kernels without uinput's UI_GET_SYSNAME this can cause misdetection of
the uinput device and test case failures. Simply picking a differently named
device avoids that.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 test/test-touchpad.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/test-touchpad.c b/test/test-touchpad.c
index 4d5fa05..570c41d 100644
--- a/test/test-touchpad.c
+++ b/test/test-touchpad.c
@@ -4614,7 +4614,7 @@ START_TEST(touchpad_disabled_double_mouse)
 	litest_assert_only_typed_events(li, LIBINPUT_EVENT_POINTER_MOTION);
 
 	mouse1 = litest_add_device(li, LITEST_MOUSE);
-	mouse2 = litest_add_device(li, LITEST_MOUSE);
+	mouse2 = litest_add_device(li, LITEST_MOUSE_LOW_DPI);
 	litest_assert_only_typed_events(li, LIBINPUT_EVENT_DEVICE_ADDED);
 
 	litest_touch_down(dev, 0, 20, 30);
@@ -4660,7 +4660,7 @@ START_TEST(touchpad_disabled_double_mouse_one_suspended)
 	litest_assert_only_typed_events(li, LIBINPUT_EVENT_POINTER_MOTION);
 
 	mouse1 = litest_add_device(li, LITEST_MOUSE);
-	mouse2 = litest_add_device(li, LITEST_MOUSE);
+	mouse2 = litest_add_device(li, LITEST_MOUSE_LOW_DPI);
 	litest_assert_only_typed_events(li, LIBINPUT_EVENT_DEVICE_ADDED);
 
 	/* Disable one external mouse -> don't expect touchpad events */
-- 
1.8.3.1

